### ISSUE TITLE
konfig dump doesn't work on Kong 0.11.0rc1

### ISSUE THREAD

**Issue by jonathan-kosgei** [mybuilder/kongfig#86]
Running konfig dump against kong 0.11.0rc1

```
kongfig dump --host 127.0.0.1:8001 --format screen
```

Gives the error
```
TypeError: Invalid Version: 0.11.0rc1 
 TypeError: Invalid Version: 0.11.0rc1
    at new SemVer (/usr/local/lib/node_modules/kongfig/node_modules/semver/semver.js:293:11)
    at compare (/usr/local/lib/node_modules/kongfig/node_modules/semver/semver.js:566:10)
    at Function.gte (/usr/local/lib/node_modules/kongfig/node_modules/semver/semver.js:615:10)
    at parseApis (/usr/local/lib/node_modules/kongfig/lib/readKongApi.js:127:26)
    at /usr/local/lib/node_modules/kongfig/lib/readKongApi.js:49:39
    at run (/usr/local/lib/node_modules/kongfig/node_modules/core-js/modules/es6.promise.js:87:22)
    at /usr/local/lib/node_modules/kongfig/node_modules/core-js/modules/es6.promise.js:100:28
    at flush (/usr/local/lib/node_modules/kongfig/node_modules/core-js/modules/_microtask.js:18:9)
    at nextTickCallbackWith0Args (node.js:419:9)
    at process._tickCallback (node.js:348:13)
```

**Comment by merqurio** [comment]
I found that the error was due to the semantic versioning. Kong guys didn't add the necessary `-` for the release candidate to be compliant with http://semver.org/. It should be `0.11.0-rc1`.

I'm preparing Pull request

**Comment by CyExy** [comment]
Fixed in `kongfig@1.4.2`
